CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
  * `ERC721`: optimize transfers by making approval clearing implicit instead of emitting an event. ([#3481](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3481))
  * `ERC721`: optimize burn by making approval clearing implicit instead of emitting an event. ([#3538](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3538))
  * `ReentrancyGuard`: Reduce code size impact of the modifier by using internal functions. ([#3515](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3515))
+ * `ERC20FlashMint`: add `_flashFee` function for calling flashFee inside it. ([#3551](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3551))
```

Comment:
```suggestion
 * `ERC20FlashMint`: add an internal `_flashFee` function for overriding. ([#3551](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3551))
```

Reply:
did I made the correct changes?

Reply:
No, this changelog entry was describing the original contents of the PR.